src/package.xml

```diff
@@ -3393,9 +3401,12 @@
         <members>GetStartedChecklistItem.Setup_Contact</members>
         <members>GetStartedChecklistItem.Stay_Track</members>
         <members>GetStartedChecklistItem.Train_Users</members>
+        <members>GetStartedChecklistItem.What_is_NonProfit_Cloud</members>
```

Comment:
Sorry, one more little change to match the filename change from above.

Reply:
I totally missed this, sorry about that. I'll make the change, thanks!